docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,133 @@
+---
+title: CRD management in ASO
+weight: -3
+---
+
+## How does CRD management work in Azure Service Operator?
+
+ASO performs CRD management differently than many traditional operators. You might notice if you install the
+[Helm chart](../installing-from-helm) or the [YAML](../installing-from-yaml) that there are NO CRDs installed by default.
+
+This is for two reasons:
+
+1. ASO supports so many CRDs that it exceeds the maximum Helm chart size when CRDs are included, so the CRDs cannot be 
+   included in the Helm chart.
+2. ASO supports so many CRDs that it can cause problems if they're all installed onto a Kubernetes cluster with a low
+   memory limit. See [#2920](https://github.com/Azure/azure-service-operator/issues/2920) for details.
+
+For the above reasons, by default each version of the ASO operator pod embeds the CRDs inside the pod and installs them when
+the pod launches. 
+
+**No CRDs are installed by default.** New CRDs will only be installed if they are specified in the 
```

Comment:
Should we mention how to install all CRDs as default?

Reply:
We kinda do under "Best Practices" where we point out that it's a bad idea. 😁

Reply:
I've updated to call it out a bit more prominently and link to best practices.